extension-manifest-v2/webpack.config.js

```diff
@@ -153,7 +160,7 @@ module.exports = {
 				],
 			}, {
 				test: /\.(js|jsx)$/,
-				include: [SHARED_COMP_DIR, ONBOARDING_DIR, PANEL_ANDROID_DIR, PANEL_DIR, LICENSES_DIR, CONTENT_SCRIPTS_DIR],
+				include: [APP_DIR],
```

Comment:
not sure what the consequences of this change may be? wouldn't it be safer to add another dir?

Reply:
It was listing all of the subpaths of the `APP_DIR`, but not "new ones", which we added recently. It should transpile all or nothing, so I think it is quite safe.